lib/load_admin_info.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'yaml'
+
+module BCDiceAPI
+  def self.load_admin_info
+    config = {}
+
+    path = File.expand_path('../config/admin.yaml', __dir__)
+    if File.exist?(path)
+      yaml = YAML.load_file(path)
+      config = {
+        name: yaml['name'].to_s,
+        url: yaml['url'].to_s,
+        email: yaml['email'].to_s
+      }
+    end
+
+    config[:name] = ENV['BCDiceAPI_ADMIN_NAME'] if ENV['BCDiceAPI_ADMIN_NAME']
+    config[:url] = ENV['BCDiceAPI_ADMIN_URL'] if ENV['BCDiceAPI_ADMIN_URL']
+    config[:email] = ENV['BCDiceAPI_ADMIN_EMAIL'] if ENV['BCDiceAPI_ADMIN_EMAIL']
```

Comment:
```suggestion
    config[:name] = ENV['BCDICE_API_ADMIN_NAME'] if ENV['BCDICE_API_ADMIN_NAME']
    config[:url] = ENV['BCDICE_API_ADMIN_URL'] if ENV['BCDICE_API_ADMIN_URL']
    config[:email] = ENV['BCDICE_API_ADMIN_EMAIL'] if ENV['BCDICE_API_ADMIN_EMAIL']
```

環境変数は慣例に倣って大文字でお願いします